packages/lesswrong/components/community/modules/CommunityMembersFullMap.tsx

```diff
@@ -25,7 +25,7 @@ const CommunityMembersFullMap = ({classes}: {
     searchClient={getSearchClient()}
   >
     <SearchResultsMap zoom={1} className={classes.map} />
-    <Configure hitsPerPage={1000} filters="_geoloc.lat>-100" />
+    <Configure hitsPerPage={1500} existsFilters={['_geoloc']} />
```

Comment:
There are ~1400 users with a map location, so I updated this page to show them all.